src/ResourceManager/Sql/ChangeLog.md

```diff
@@ -19,6 +19,8 @@
 -->
 ## Current Release
 * Added ability to rename database using Set-AzureRmSqlDatabase
+* Fixed issue https://github.com/Azure/azure-powershell/issues/4974
+	- Removed invalid AUDIT_CHANGED_GROUP value from auditing cmdlets
```

Comment:
@cormacpayne, @markcowl  Do I still need to include this?

Reply:
@ranisha2 I would change the second line to say "Providing invalid AUDIT_CHANGED_GROUP value for auditing cmdlets no longer throws an error and will be removed in an upcoming release."

Reply:
@cormacpayne Done